### PR TITLE
docs: update readme

### PR DESCRIPTION
**Pull request by imsobear**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,47 +2,73 @@ English | [简体中文](https://ice.work/docs/icestark/about)
...
 
-```bash
-npm install @ice/stark --save
-```
+- No framework constraint for main&sub applications, Support React/Vue/Angular/...
+- Sub-application support multiple types of entry: js&css, html entry, html content
+- JavaScript sandbox by `Proxy` API
 
-## Documentation
+## Showcase 🎃
```
Comment (maoxiaoke) [inline README.md:15]: showcases ?

File: README.md [hunk 2]
```diff
@@ -2,47 +2,73 @@ English | [简体中文](https://ice.work/docs/icestark/about)
...
 
-## Installation
+## Features 🎉
 
-```bash
-npm install @ice/stark --save
-```
+- No framework constraint for main&sub applications, Support React/Vue/Angular/...
+- Sub-application support multiple types of entry: js&css, html entry, html content
+- JavaScript sandbox by `Proxy` API
```
Comment (maoxiaoke) [inline README.md:13]: - Simple and completed solution of micro frontend, all  in one ?

File: README.md [hunk 3]
```diff
@@ -2,47 +2,73 @@ English | [简体中文](https://ice.work/docs/icestark/about)
...
 
 [![NPM version](https://img.shields.io/npm/v/@ice/stark.svg?style=flat)](https://npmjs.org/package/@ice/stark) [![Package Quality](https://npm.packagequality.com/shield/@ice%2Fstark.svg)](https://packagequality.com/#?package=@ice%2Fstark) [![build status](https://img.shields.io/travis/ice-lab/icestark.svg?style=flat-square)](https://travis-ci.org/ice-lab/icestark) [![Test coverage](https://img.shields.io/codecov/c/github/ice-lab/icestark.svg?style=flat-square)](https://codecov.io/gh/ice-lab/icestark) [![NPM downloads](http://img.shields.io/npm/dm/@ice/stark.svg?style=flat)](https://npmjs.org/package/@ice/stark) [![David deps](https://img.shields.io/david/ice-lab/icestark.svg?style=flat-square)](https://david-dm.org/ice-lab/icestark)
 
-## Installation
+## Features 🎉
 
-```bash
-npm install @ice/stark --save
-```
+- No framework constraint for main&sub applications, Support React/Vue/Angular/...
```
Comment (maoxiaoke) [inline README.md:11]: Support -> support

File: README.md (hunk 2 above)
Comment (imsobear) [reply README.md:14]: 其他都改了，这个先不加吧

**Review by maoxiaoke — APPROVED** [review]

**Pull request merged**